src/VisualStudio/Core/Next/FindReferences/StreamingFindReferencesPresenter.TableDataSourceFindReferencesContext.cs

```diff
@@ -470,8 +525,6 @@ private TextSpan GetRegionSpanForReference(SourceText sourceText, TextSpan refer
                     result.Add(span);
                     startPosition = span.TextSpan.End;
                 }
-
-                return result;
```

Comment:
Does this not need to fill in a span between the last classified span and the end of widenedSpan from above? Why the asymmetry?